### ISSUE TITLE
Note reading issue

### ISSUE THREAD

**Issue by rigo** [pump-io/pump.io#813]
After logging in on identi.ca, a note notification is displayed bellow the envelope saying: "a person posted a note". However, when I try to read it, the message "Error fetching notice" is displayed. If refresh the page I get a different error message: "No such note: 84945130". If I go to activity page, the notification no longer appears, and the counter is correctly updated. However, when I log back in, the notification appears again and the counter reverts to the previous state.

**Comment by strugee** [comment]
@jankusanagi do you know if there's still stuff that should be done to production identi.ca data to fix this? I'm thinking maybe we should close

**Comment by jankusanagi** [comment]
@strugee I don't know if there is a solution, but it would probably not be simple.
Personally, I doubt it's worth it. It's the kind of thing that only people who never use their account notice, when they log in and find their 4 year old notifications pointing to the old StatusNet URLs.

I don't know if @evanp intends to map those URLs to something on identi.ca specifically, but it's not a general Pump.io issue AFAIK. It's not even an issue for current identi.ca, just for the really old stuff.

**Comment by strugee** [comment]
@jankusanagi right, I was mostly just wondering if Evan did something and forgot to close this ticket.

**Comment by strugee** [comment]
\*shrug\*
